llama-index-integrations/graph_stores/llama-index-graph-stores-falkordb/llama_index/graph_stores/falkordb/base.py

```diff
@@ -31,7 +32,12 @@ def __init__(
         self._node_label = node_label
 
         self._driver = FalkorDB.from_url(url).select_graph(database)
-        self._driver.query(f"CREATE INDEX FOR (n:`{self._node_label}`) ON (n.id)")
```

Comment:
Couldn't we instead query if the index exists already? Seems a little safer (no worries if not though)

Reply:
@noahxzhu any thoughts on logan's suggestion above?

Reply:
@nerdai @logan-markewich did an update and tested, this should be fine for most cases, but not elegant, I'm not so familiar with falkordb.